lib/nora/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Nora
-  VERSION = "0.2".freeze
+  VERSION = "0.3"
```

Comment:
Bumping a new version so users (a.k.a. me) can `gem update` and things will work.